src/target/source/codegen_source_base.h

```diff
@@ -52,6 +52,18 @@ class CodeGenSourceBase {
    * \param value The constant value.
    */
   void MarkConst(std::string value);
+  /*!
+   * Print Type represetnation of type t.
+   * \param t The type representation.
+   * \param os The stream to print the ctype into
+   */
+  virtual void PrintType(DataType t, std::ostream& os);  // NOLINT(*)
+  /*!
+   * Print Type represetnation of type type.
+   * \param type The type representation.
```

Comment:
Can we use the same names in both of these blocks? i.e. `type type` or `type t` ? 

Also minor typo:
```suggestion
   * Print Type representation of type type.
   * \param type The type representation.
```

Reply:
Done